CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+## Setup
+
+```bash
+yarn install
```

Comment:
Let's say `npm install` here – people who know what yarn is will use it anyway, and those who don't aren't confused wtf yarn is!